models/routineexercise.js

```diff
@@ -3,11 +3,11 @@ const {
   Model
 } = require('sequelize');
 
-const Excercise = require('./excercise');
+const Excercise = require('./exercise');
 const Routine = require('./routine');
 
 module.exports = (sequelize, DataTypes) => {
-  class RoutineExcercises extends Model {
+  class RoutineExercise extends Model {
```

Comment:
Rename to Routine

Reply:
There is already Routine model and table, **RoutineExercises** is the pivot table because there is many to many relationship between **Routine** and **Exercise**